torch/_inductor/codegen/triton.py

```diff
@@ -1062,7 +1062,13 @@ def abs(x):
 
     @staticmethod
     def truediv(x, y):
-        out = f"({x} / {y})"
+        x_dtype = getattr(x, "dtype", None)
+        y_dtype = getattr(y, "dtype", None)
+
+        if x_dtype == torch.float32 and y_dtype == torch.float32:
+            out = f"triton.language.div_rn({x}, {y})"
+        else:
+            out = f"({x} / {y})"
```

Comment:
can you leave a comment that `x /y` in triton is lowered to `dif.full` which is approximate, instead of `div_rn`